main/ertp/api/issuer.md

```diff
@@ -1,214 +1,275 @@
 # Issuer
 
-The issuer cannot mint new amounts, but it can create empty purses and payments. The issuer can also transform payments (splitting payments, combining
-payments, burning payments, and claiming payments exclusively). The issuer should be obtained from a trusted source and then relied upon as the
-authority as to whether an untrusted payment is valid.
+An `issuer` is the authority on what holds digital assets of its kind. 
+While it cannot create new value by creating digital assets like a `mint`, 
+it recognizes and identifies `purses` and `payments` that carry actual value.
+It can create empty `purses` and transform `payments` (by splitting, 
+combining, burning, or exclusively claiming). 
+
+An `issuer` has an unchangeable one-to-one relationship with the `mint` and
+`brand` that were created with it. For any `brand` for which you will accept
+`payments` in, you should obtain its `issuer` from a trusted source.
+You can then rely on that `issuer` as the authority to 
+determine if an untrusted `payment` of that `brand` is valid.
+
+## makeIssuerKit(allegedName, amountMathKind)
+- `allegedName` `{String}` 
+- `amountMathKind` `{MathKind}` - optional
+- Returns `{IssuerKit}`
+
+Makes a new `issuer` as well as its one-to-one associated ERTP objects; a `mint` and a `brand`. 
+All are in unchangeable one-to-one relationships with each other. 
+
+While this also returns an `amountMath`, we strongly urge that instead of using that one, which
+is asynchronous, you use `makeLocalAmountMath(issuer)` to create a local, synchronous, version
+and use that one. 
+
+The `allegedName` is available from the `brand` or the `issuer` in asset descriptions. The `allegedName` is 
+useful for debugging and double-checking assumptions, but should not be trusted.
+
+The optional `amountMathKind` specifies the kind of math to use with the digital assets. 
+Each implements all of the same set of API methods (i.e. `amountMath` methods are 
+polymorphic). We recommend you import the `MathKind` values from `@agoric/ERTP` 
+instead of making the strings yourself.
```

Comment:
```suggestion
polymorphic). We recommend you import the `MathKind` values from `@agoric/ERTP` 
instead of using strings.
```
I don't think JavaScript programmers think of using 'foo' as  making a string, though that' s a fair description of the effect.